frontend/app/src/entities/nodes/object/ui/object-table/cells/node-kind-cell.tsx

```diff
@@ -0,0 +1,14 @@
+import { useSchema } from "@/entities/schema/ui/hooks/useSchema";
+import { Badge } from "@/shared/components/ui/badge";
+
+export function NodeKindCell({ kind }: { kind: string }) {
+  const { schema } = useSchema(kind);
+
+  if (!kind || !schema) return "-";
+
+  return (
+    <div className="flex items-center gap-2">
+      {schema?.label} <Badge>{schema?.namespace}</Badge>
+    </div>
```

Comment:
Checking `!schema` is enough. No need to check for `!kind`.
Also, you do not need optional chaining (?.) because you check `schema` exists before.

Finally the name indicates it's a cell, whereas it's a component that format node_kind for TableAttributeCell